packages/wallet-service/tests/env.test.ts

```diff
@@ -0,0 +1,25 @@
+import config, { loadEnvConfig } from '@src/config';
+
+test('Configuration should load correctly during tests', () => {
+  expect.hasAssertions();
+
+  const loadedConfig = loadEnvConfig();
+  expect(loadedConfig).toStrictEqual(config);
```

Comment:
Since you're not mocking `confirmFirstAddress`, this will depend on the `.env`, right?

Maybe just set it

`process.env.CONFIRM_FIRST_ADDRESS = 'true'`